### PR TITLE
feat: allow login to non-db contracts

### PR DESCRIPTION
**Pull request by gagdiez**
Before this commit, each time updateAccount was triggered the VM checked if the selector was connected - i.e. had an access key - to a contract different from SocialDB, and proceed to logout the user if that was the case.

This commit removes this constraint, allowing users to connect to both SocialDB and other contracts.

### CODE REVIEW

**Comment by gagdiez** [comment]
this PR solves issue #184

**Comment by evgenykuzyakov** [comment]
Don't remove the change, but make it configurable instead. This is desired behavior for a default gateways that only use it with a single contract

**Comment by gagdiez** [comment]
@evgenykuzyakov yes, that makes sense. I reverted my changed and added a check for a config flag.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## TBD
+- Made optional to logout the wallet selector when connecting to a contract different from `config.contractName`
+
+  - To enable this feature, pass the allowOtherContracts flag to the config parameter of initNear: `initNear({networkId, selector, config: { allowOtherContracts: true }} )`
```
Comment (charleslavon) [inline CHANGELOG.md]:
```suggestion
  - To enable this feature, pass the allowOtherContracts flag to the config parameter of initNear: `initNear({networkId, selector, allowOtherContracts: { contractId: 'example.testnet' }} )`
```

**Comment by gagdiez** [comment]
changed the code based on @charleslavon request, now the `initNear` takes a `allowExternalContract: <contractId>`

**Comment by charleslavon** [comment]
I mean to update this PR, not dev. Will push another change shortly.

**Comment by charleslavon** [comment]
I meant to push to this pr, not merge to dev. I'll push another change shortly.

**Comment by charleslavon** [comment]
https://github.com/NearSocial/VM/pull/198

**Pull request merged**